Il2CppInspector.GUI/MainWindow.xaml.cs

```diff
@@ -196,6 +197,14 @@ public partial class MainWindow : Window
 
             // Populate TreeView with namespace hierarchy
             trvNamespaces.ItemsSource = namespaceTree;
+
+            var prevSelection = cboUnityVersion.SelectedItem;
+            cboUnityVersion.Items.Clear();
+            foreach (var version in UnityHeader.GuessHeadersForModel(model))
+                cboUnityVersion.Items.Add(version);
+            cboUnityVersion.SelectedIndex = 0;
+            if (prevSelection != null)
+                cboUnityVersion.SelectedItem = prevSelection;
```

Comment:
Is it possible to add an auto-detect option to the combo-box similar to what the CLI can do and make it the default? If they pick the wrong version will everything succeed but just produce the wrong type layouts in the IDAPython file?